aiorussound/const.py

```diff
@@ -15,6 +15,8 @@
 
 MAX_SOURCE = 17
 
+SYSTEM_VARIABLES = "System"
```

Comment:
I might consider using something like `SYSTEM_KEY` to be more clear than variables.